sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -450,12 +450,14 @@ class Dataset[T] private[sql](
   /**
    * Returns a new Dataset where each record has been mapped on to the specified type. The
    * method used to map columns depend on the type of `U`:
-   *  - When `U` is a class, fields for the class will be mapped to columns of the same name
-   *    (case sensitivity is determined by `spark.sql.caseSensitive`).
-   *  - When `U` is a tuple, the columns will be mapped by ordinal (i.e. the first column will
-   *    be assigned to `_1`).
-   *  - When `U` is a primitive type (i.e. String, Int, etc), then the first column of the
-   *    `DataFrame` will be used.
+   * <ul>
+   *   <li>When `U` is a class, fields for the class will be mapped to columns of the same name
```

Comment:
Nit: close `<li>` tag

Reply:
BTW... doesn't scaladoc interpret markdown tags? Is it just that these didn't have a blank line above or something? I don't know it well.

Reply:
nice catch!thanks! the scaladoc supports wiki-style syntax and HTML, not markdown.

Reply:
Oh OK. Maybe I'm crazy but I thought `- foo` style lists work, but remember that sometimes they don't unless there's a blank line above it. I'm just wondering if that's the issue here. Or if it's not -- there are many more instances like this in the code, no? let's fix more of them if it's necessary